rafs/src/metadata/direct_v6.rs

```diff
@@ -1159,14 +1160,49 @@ impl RafsInode for OndiskInodeWrapper {
         match self.parent_inode.get() {
             Some(parent) => parent,
             None => {
-                debug_assert!(self.is_dir());
```

Comment:
It would be better to keep `parent()` as is, and modify `nydus-image` to call `file_parent_inode()` directly.

Reply:
Ok, i got it.